File: src/main/java/br/org/soujava/domain/Slide.java
package br.org.soujava.domain;

import java.io.Serializable;

import lombok.Builder;
import lombok.Data;

/**
 * Slide of the Website SouJava
 * 
 * @author Fernando Arruda - progfer@gmail.com
 *
 */
@Data
@Builder
public class Slide implements Serializable{

	private String id;
	private String title;
	private String description;
	private boolean disabled;
	private int order;
	
}

File: src/main/angular/src/app/slide/slide.ts
export class Slide{

  id: string;
  title: string;
  description: string;
  disabled: boolean;
  order: number;

  constructor(id:string, title:string, description:string, disabled:boolean, order:number){
    this.id = id;
    this.title = title;
    this.description = description;
    this.disabled = disabled;
    this.order = order;
  }

}
